build/custom.mk

```diff
@@ -1,9 +1 @@
-## Runs errcheck against all packages.
-.PHONY: errcheck
-errcheck:
```

Comment:
Can you please elaborate why you are removing this? It runs fine on CI.

Reply:
I get this running `make golint` locally, like I said, I did not find a way to change the flags to ignore these. If it works on ci, I am assuming these warnings are disabled somehow?
```
levbrouk@Levs-MacBook-Pro:mattermost-plugin-gitlab (lev-GH-107-errcheck)$ make golint
Running lint
env GO111MODULE=off /Users/levbrouk/go/bin/go get golang.org/x/lint/golint
/Users/levbrouk/gopath/bin/golint -set_exit_status ./...
server/api.go:23:2: don't use ALL_CAPS in Go names; use CamelCase
server/api.go:23:2: exported const API_ERROR_ID_NOT_CONNECTED should have comment (or a comment on this block) or be unexported
server/api.go:24:2: don't use ALL_CAPS in Go names; use CamelCase
server/api.go:27:6: exported type APIErrorResponse should have comment or be unexported
server/api.go:255:6: exported type ConnectedResponse should have comment or be unexported
server/api.go:264:6: exported type GitlabUserRequest should have comment or be unexported
server/api.go:268:6: exported type GitlabUserResponse should have comment or be unexported
server/command.go:12:7: don't use ALL_CAPS in Go names; use CamelCase
```

Reply:
`golint` is an optional target. We don't run it in CI.

But what you are removing here is `errcheck` which we run in CI and wich also works fine.

Reply:
@hanzei ping? This is for 5.16.

Reply:
Running `errcheck` produces failures. Does it properly support go modules? (running it without, it's failing to locate go-gitlab)

Reply:
Would you please post the error message here?